.gitmodules

```diff
@@ -1,9 +1,9 @@
 [submodule "loader"]
 	path = external/OpenCL-ICD-Loader
-	url = https://github.com/KhronosGroup/OpenCL-ICD-Loader.git
+	url = https://github.com/StreamHPC/OpenCL-ICD-Loader.git
 [submodule "headers"]
 	path = external/OpenCL-Headers
-	url = https://github.com/KhronosGroup/OpenCL-Headers.git
+	url = https://github.com/StreamHPC/OpenCL-Headers.git
 [submodule "headers-cpp"]
 	path = external/OpenCL-CLHPP
-	url = https://github.com/KhronosGroup/OpenCL-CLHPP.git
+	url = https://github.com/StreamHPC/OpenCL-CLHPP.git
```

Comment:
Is this change to the StreamHPC repos intended?  Should this be reverted before merging?

Reply:
Yes, these should be reverted before merging and are/were only needed while downstream changes were required for clean builds.